html_extra/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Disallow:
+sitemap: https://my-site.com/docs/sitemap.xml
```

Comment:
needs to be refactored to have multiple sitemaps for older versions

Reply:
What we could do is running `make dirhtml` for all previous builds and create a sitemap retrospectively for each.
Also, to not add the sitemaps manually to the sitemap index, we should use our `cd.yml` script to add the sitemaps to the robots.txt (https://developers.google.com/search/docs/advanced/sitemaps/large-sitemaps?hl=en&visit_id=637500524312622086-433016932&rd=1)